addons/membership/tests/test_membership.py

```diff
@@ -60,7 +60,7 @@ def test_paid_membership(self):
         self.assertEqual(
```

Comment:
at line [#31](https://github.com/odoo/odoo/pull/8141/files#diff-c34058894ae4adbf78c3b9441bcfd107R31), remove index [0] . limit=1 is already there.